addons/mail/models/mail_template.py

```diff
@@ -238,6 +242,28 @@ def onchange_sub_model_object_value_field(self):
             self.null_value = False
 
     @api.multi
+    def write(self, vals):
+        # This override is for checking that the template is correct when
+        # saving it
+        test_model = 'res.partner'
```

Comment:
Don't you think this will cause issues ?

Reply:
In what sense? self.model could be used as well, we just need a random record to test that the rendering doesn't crash, doesn't matter which model it's from (I think)

Reply:
Well if you look at the template model it is meant to run on any kind of model. However content may be model dependant. Let us imagine a template on event.registration model. Don't you think you may go through false issues when running it on a res.partner model ?

Reply:
You're right, I thought bad variables were replaced by strings and didn't raise any errors so I thought that'd be fine, but actually I guess mako uses a javascript-like syntax where ${object.name.poop} works (undefined) but ${object.name.poop.lol} raises an error.

I'll change this to use self.model or vals['model_id'] if the model's been modified